deeplearning4j/deeplearning4j-modelimport/src/test/java/org/deeplearning4j/nn/modelimport/keras/layers/convolution/KerasConvolution3DTest.java

```diff
@@ -145,7 +151,7 @@ private void buildConvolution3DLayer(KerasLayerConfiguration conf, Integer keras
         config.put(conf.getLAYER_FIELD_BORDER_MODE(), BORDER_MODE_VALID);
         layerConfig.put(conf.getLAYER_FIELD_CONFIG(), config);
         layerConfig.put(conf.getLAYER_FIELD_KERAS_VERSION(), kerasVersion);
-        ConvolutionLayer layer = new KerasConvolution3D(layerConfig).getConvolution3DLayer();
+        Convolution3D layer = (Convolution3D) new KerasConvolution3D(layerConfig).getConvolution3DLayer();
```

Comment:
Is that cast still necessary here?

Reply:
It was returning a ConvolutionLayer. Probably not now.